FaceID/DB_faceID.py

```diff
@@ -280,16 +294,66 @@ def searchFace():
                 minSize = (int(minW), int(minH)),
             )
 
-            for(x,y,w,h) in faces:
+            for(x,y,w,h) in faces: # detects face
 
-                EmpID, confidence = recognizer.predict(gray[y:y+h,x:x+w])
+                EmpID, confidence = recognizer.predict(gray[y:y+h,x:x+w]) # recognizes face
+              
                 # Checks confidence threshold  
                 if ((confidence < 100) and (100-confidence)>25):
+
                     msg = str(EmpID)
                     confidence = "{0}%".format(round(100 - confidence))
-                    cv2.circle(img, (int(x+(w/2)),int(y+(h/2))), 108, (50,0,0), 3)
+                    cv2.circle(img, (int(x+(w/2)),int(y+(h/2))), 108, (0,220,0), 3)
                     cv2.putText(img, "EmpID: "+msg, (x+w+25,int(y+(h/2))), font, 1, (255,255,0), 2)
                     cv2.putText(img, "conf: "+str(confidence), (x+w+25,int(y+(h/2))+30), font, .75, (255,255,0), 2)
+         
+                    for person in photoRegister:
+                        extractEmpID = (person[1].split("_"))[0]
+
+                        if (int(extractEmpID) == EmpID):
+
+                            lastScan = list(map(int,person[0].split(":")))
+                            currentTime = datetime.utcnow().time()
+                            timeNow = (str(currentTime.hour) + ":" + str(currentTime.minute) + ":" + str(currentTime.second)).split(":")
+                            timeNow = list(map(int,timeNow))
+
+                            hourDiff = timeNow[0] - lastScan[0]
+                            minDiff = timeNow[1] - lastScan[1]
+                            secDiff = timeNow[2] - lastScan[2]
+                            
+                            if hourDiff<0: hourDiff += 24
+                            if minDiff<0: minDiff += 60
+                            if secDiff<0: secDiff += 60
```

Comment:
These three line should have spaces around the `<` for consistency and readability

Reply:
Fixed spacing and date threshold management